### PR TITLE
Remove six decorator for unicode text conversion

### PR DESCRIPTION
**Pull request by jborean93**
##### SUMMARY
Removes the six unicode decorator and unify our text handling to to_text.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
collection

### CODE REVIEW

**Comment by abadger** [comment]
+1 from me

File: lib/ansible/galaxy/collection.py [hunk 1]
```diff
@@ -82,7 +81,10 @@ def __init__(self, namespace, name, b_path, source, versions, requirement, force
         self.add_requirement(parent, requirement)
 
     def __str__(self):
-        return to_text("%s.%s" % (self.namespace, self.name))
+        return to_native("%s.%s" % (self.namespace, self.name))
+
+    def __unicode__(self):
+        return u"%s.%s" % (self.namespace, self.name)
```
Comment (webknjaz) [inline lib/ansible/galaxy/collection.py:87]: maybe
```suggestion
        return to_text(str(self))
```

cc @abadger?

**Comment by abadger** [comment]
No because we're supposed to be following the unicode sandwich conventions
in our code.  All variables are made text when they enter the system and
turned into bytes (if necessary) when they leaves the system.  If name or
namespace are not text here, then the to_text call belongs where the bytes
value is being assigned to name or namespace rather than here.

**Comment by jborean93** [comment]
Test failure in unrelated to the changes.

**Pull request merged**